test/integration/install/install_uninstall_test.go

```diff
@@ -0,0 +1,202 @@
+package install
+
+import (
+	"context"
+	"io"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/shirou/gopsutil/process"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/compose"
+)
+
+const (
+	osReleasePath  = "/etc/os-release"
+	maxFileSize    = int64(20000000)
+	maxInstallTime = 30 * time.Second
+)
+
+var (
+	AGENT_PACKAGE_FILENAME = os.Getenv("PACKAGE_NAME")
+	AGENT_PACKAGE_FILEPATH = "/agent/build/" + AGENT_PACKAGE_FILENAME + ".deb"
```

Comment:
Should we hardcode the package extension? This might not always run on a Debian or Ubuntu environment.

Reply:
I should've stated in this PR that the scope was solely ubuntu as a starting point.

However this was easy to support so I decided to include in: https://github.com/nginx/agent/pull/147/commits/bb3607a7d4d2386822297270586274f800e8b4f7